diff --git a/cactus-react-native.podspec b/cactus-react-native.podspec
index 5225f9d6af9eb7fc68653edec7bac1c134d83ad3..4cceae4d71850215baf92f81d804310a5c61d43b 100644
--- a/cactus-react-native.podspec
+++ b/cactus-react-native.podspec
@@ -32,7 +32,7 @@ Pod::Spec.new do |s|
   s.pod_target_xcconfig = {
     "OTHER_LDFLAGS" => base_ld_flags,
     "OTHER_CFLAGS" => base_optimizer_flags,
-    "OTHER_CPLUSPLUSFLAGS" => base_optimizer_flags + " -std=c++17"
+    "OTHER_CPLUSPLUSFLAGS" => base_optimizer_flags + " -std=c++20"
   }
 
   # Don't install the dependencies when we run `pod install` in the old architecture.
