diff --git a/src/index.ts b/src/index.ts
index 24e4a80f928d89c6d9ab9011c37145e64084605d..0577cb7fa9070d8331865b399cdfbb0aee4f2e6d 100644
--- a/src/index.ts
+++ b/src/index.ts
@@ -2,7 +2,6 @@
 
 import './publicGlobals';
 
-import * as Animated from './Animated';
 import { initializeReanimatedModule } from './initializers';
 import { ReanimatedModule } from './ReanimatedModule';
 
@@ -10,6 +9,9 @@ import { ReanimatedModule } from './ReanimatedModule';
 // universal source of truth for it.
 initializeReanimatedModule(ReanimatedModule);
 
+// eslint-disable-next-line import/first
+import * as Animated from './Animated';
+
 export default Animated;
 
 export type {
diff --git a/src/initializers.ts b/src/initializers.ts
index 26cc7a992c7610c2a72cdec0110becbe2efe82da..f56c41b3cba504fe432fb777383c44d1224ebc72 100644
--- a/src/initializers.ts
+++ b/src/initializers.ts
@@ -4,6 +4,7 @@ import { executeOnUIRuntimeSync } from 'react-native-worklets';
 import {
   DEFAULT_LOGGER_CONFIG,
   IS_WEB,
+  ReanimatedError,
   registerLoggerConfig,
   SHOULD_BE_USE_WEB,
 } from './common';
